element-connector/pom.xml

```diff
@@ -65,7 +65,6 @@
 			<plugin>
 				<groupId>org.apache.maven.plugins</groupId>
 				<artifactId>maven-jar-plugin</artifactId>
-				<version>3.0.2</version>
```

Comment:
if we remove the version here, we should declare it in the top level POM in  `build/pluginManagement`. Otherwise we might use different versions of the plugin in different builds.